src/main/scala/molmed/config/FileAndProgramResourceConfig.scala

```diff
@@ -263,6 +209,15 @@ trait FileAndProgramResourceConfig {
       if (this.snpEffPath == null)
         this.snpEffPath = getFileFromKey(programNameToPathsMap, Constants.SNP_EFF)
 
+      if (this.novoalignCS == null)
```

Comment:
I'm not sure, but maybe this should be moved into a separate trait. If you are planing to use write your own QScripts that would create more separation and less confusion for the end user since they don't have to be presented with options for lots of programs that they are not going to use..